app/assets/stylesheets/variables.scss

```diff
@@ -36,3 +36,6 @@ $accordion-button-focus-box-shadow: 0 0 0.25rem 0.25rem rgba($sonic-silver, 0.25
 
 // Table
 $table-border-color: $silver-sand;
+
+// Fonts
+$font-family-sans-serif: 'Source Sans Pro', sans-serif;
```

Comment:
This is the font specified in the Stanford design system: https://decanter.stanford.edu/page/brand-design-elements-typography/

Proof here: https://decanter.stanford.edu/component/identity-global-footer/